modules/exploits/linux/http/f5_icontrol_rce.rb

```diff
@@ -0,0 +1,81 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Undisclosed requests may bypass iControl REST authentication',
```

Comment:
Module names must include the software name, and optionally include the vulnerable component/parameter/functionality. and usually approximately matches the file name.

Suggest something like this:

* F5 BIG-IP iControl REST Command Execution
* F5 BIG-IP iControl REST Authenticated Command Execution

Reply:
More like `F5 BIG-IP iControl REST Unauthenticated Command Execution` then :P

Reply:
2 iControl 2 Furious, please.